Mixpanel/InAppNotifications.swift

```diff
@@ -79,20 +82,32 @@ class InAppNotifications: NotificationViewControllerDelegate {
     }
 
     @discardableResult
-    func notificationShouldDismiss(controller: BaseNotificationViewController, callToActionURL: URL?) -> Bool {
+    func notificationShouldDismiss(controller: BaseNotificationViewController,
+                                   callToActionURL: URL?,
+                                   shouldTrack: Bool,
```

Comment:
could we have a default value?

Reply:
i think i tried to put one but since it's being used outside this file from the protocol definition, it wouldn't let me omit it so the default value wasn't useful